### DISCUSSION TITLE
VSCode Version Update Crippled Prettier for APEX Classes

### DISCUSSION THREAD

**Discussion by bdaul-trail** [microsoft/vscode-discussions#1624] [Extension Development QnA]
Version 1.91.1 of Visual Studio Code appears that it was the last version to work with APEX formatting using Prettier.

Is there a solution for this?  

A downgrade of VSCode to version 1.91.1 and Prettier does work as expected.  

The problem is that I have VSCode Settings for Update:Mode set to 'none' and it is ignored.  

When I open VSCode it is upgraded to version 1.93.0 and Prettier breaks again.

Prettier formatting is required to move through the approval process. 

If anyone has a solution could you please offer it?  

Thanks

**Comment by gjsjohnmurray** [comment]
Please raise the Prettier issue with the extension authors, and report the auto update issue to the main VS Code repository. The Report Issue option from the Help menu can help you do both of these. This discussion forum is for authors of extensions.

**Comment by olguzzar** [comment]
Duplicate of https://github.com/microsoft/vscode-discussions/discussions/1626
